pacman-nodejs-app/tools/migrate/kmt.sh

```diff
@@ -174,6 +174,11 @@ function save_src_cluster_resources {
     # Save off public IP addresses for services in source cluster
     for s in ${services}; do
         eval ${s^^}_SRC_PUBLIC_IP=$(kubectl get service ${s} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+	# Check for hostname (used by aws, for example) or ip (used by gke, for example)
+	src_ip_var=$(echo ${s^^}_SRC_PUBLIC_IP)
+	if [[ ${!src_ip_var} == '' ]]; then
+        	eval ${s^^}_SRC_PUBLIC_IP=$(kubectl get service ${s} -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+        fi
```

Comment:
yeah I installed some go formatting for vim and it's kind of taken over. . .will fix this.